src/common/components/keyword/keyword.module.scss

```diff
@@ -10,6 +10,24 @@
   padding: 0 var(--spacing-s);
   text-decoration: none;
 }
+.keyword-red {
+  background-color: var(--color-brick);
+  color: white;
+}
+
+.keyword-green {
+  background-color: var(--color-copper-dark);
+  color: white;
+}
```

Comment:
Are the keyword colors actually categories that have nothing to do with color? I mean are they actually semantic categories, not color categories? If they are color categories then should the color of the category's name be reflected in the actual color shown? We could have a chat about this too, maybe easier to talk live / throw ideas around.

Reply:
If they're semantic categories then I'd say renaming the categories to their semantic meanings would be better than having the semantic information mapped to color in the name and still mapped to some other color in the visible shown color, if I understood correctly what's going on here. But let's talk!

Reply:
I thought about this and I personally came to conclusion that the color-as-a-identifier told me more than semantic. I can think that green is a good thing and there is something good for me, but if the name is something like "free", it does not tell me the color of it and I might not even understand the whole "labeling". But, if this is something that really matters to you and you come up with better naming, it's OK for me to change the implementation. I think this is very trivial thing after all.

Reply:
Also, this styling is now in keyword context, not in enrolment status context.

Reply:
We had a talk and this is ok.